assets/scss/drupal/module/_civiremote_funding.scss

```diff
@@ -17,3 +17,12 @@ $legend-font-size: 1.5rem;
 #funding-form fieldset fieldset fieldset legend {
   font-size: $legend-font-size * .85;
 }
+
+// Reduce required space. (There's still a bottom margin.)
+#funding-form {
+  .details-wrapper, .fieldset-wrapper, .json-forms-array-wrapper {
```

Comment:
Let's move every selector to a new line, please.

Reply:
Changed.